training/deepspeech_training/util/augmentations.py

```diff
@@ -349,8 +349,12 @@ def apply(self, sample, clock=0.0):
         audio = sample.audio
         orig_len = len(audio)
         audio = np.swapaxes(audio, 0, 1)
-        audio = resample(audio, sample.audio_format.rate, rate)
-        audio = resample(audio, rate, sample.audio_format.rate)
+        if audio.shape[0] < 2:
```

Comment:
nit: can you add a comment explaining and documenting that it's librosa api change and maybe quickly describing the involved shapes?